Day2/BoxFactory.py

```diff
@@ -0,0 +1,30 @@
+class BoxFactory:
+    
+    packaging_dimensions = None
+    finished_packaging = []
+    
+    def __init__(self, packaging_dimensions):
+        self.packaging_dimensions = packaging_dimensions
+        self.finished_packaging = []
+
+    def make_box(self, length, width, height):
+        side1 = length*width
+        side2 = width*height
+        side3 = height*length
+        return (side1, side2, side3)
+    
+    def make_all_boxes(self):
+        for box in self.packaging_dimensions:
+            box_with_sides = self.make_box(*box)
+            self.finished_packaging.append(box_with_sides)
```

Comment:
finished PACKAGING nem tart boxokat
finished BOXES tart boxot 

igen a doboz az egyfele packaging DE csak dobozt teszel bele, a doboz classon belul vagy itt nem lesz mas tipusu packaging itt BOX IS GOD

nem forditasz :D nem kell szoismetlesre figyelni, pontossagra kell! 1 dolognak 1 neve van nem tobb